docker-compose/docker-compose-v2.1.yaml

```diff
@@ -4,7 +4,7 @@ services:
   unifiedpushserver:
     image: docker.io/aerogear/unifiedpush-wildfly:1.3.0
     volumes:
-       - ./helper:/ups-helper
+       - ./helper:/ups-helper:z
```

Comment:
what does the `z` at the end mean?

Reply:
SELinux, for safer mounts, on linux 
(this setup - as you now know, anyway works on Linux, unless you do some fancy networking in docker)